Create README.md

May-Lee opened the pull request:
Bare bones Readme so that anyone arriving at this page will see something. Will be added to as needed.

May-Lee committed:
Create README.md

Bare bones Readme so that anyone arriving at this page will see something. Will be added to as needed.

exarkun reviewed (approved):
Great.  Thanks!

I like https://rhodesmill.org/brandon/2012/one-sentence-per-line/ for formatting documentation.  If it appeals to you then perhaps docs in this repo could use it.  If not, that's fine with me too.

The link into the Code of Conduct in the Tahoe-LAFS repo bothers me just the tiniest possible amount.  I wouldn't want someone to re-arrange the Tahoe-LAFS docs and break this link.  However, the chances of that seem small enough and the fix is easiest enough that I also don't want to spend a lot of effort coming up with a solution.  So it's probably fine as-is.

Please merge when you're happy with it. :)

May-Lee committed:
Update README.md

Okay, thanks @exarkun. I understand about the broken link thing. Ideally, I would like to add a link checker to work on my CI chops but maybe not today ;)
